src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
```

Comment:
After running the create-react-app it would be a good idea to delete the default styles from app.css. Otherwise you could have some css conflicts between your custom css and the css from create-react-app 💇🏽‍♂️